rule/rule.go

```diff
@@ -97,6 +102,16 @@ func LoadWorkspaceFile(path, pkg string) (*File, error) {
 	return LoadWorkspaceData(path, pkg, data)
 }
 
+// LoadMacroFile is similar to LoadFile but parses the file as a bzl
```

Comment:
Please expand the doc here. It's important to say that this parses rules out of a Starlark function. `Sync` and `Save` calls will update the body of the function.